src/tlo/methods/rti.py

```diff
@@ -4800,6 +4814,11 @@ def apply(self, person_id, squeeze_factor):
             # RTI_Med
             assert df.loc[person_id, 'rt_diagnosed'], 'This person has not been through a and e'
             assert df.loc[person_id, 'rt_med_int'], 'This person has not been through rti med int'
+
+            # TODO: @Joe to confirm major surgery pkg should be used here (as the original set of equipment here was not
+            #  the full pkg)
```

Comment:
comment resolved?

Reply:
Not resolved, I forgot to ask @joehcollins about this. Or @andrew-phillips-1 can reslove, if he knows.

Reply:
decided to wait for @joehcollins to confirm